recipe/install-gcc.sh

```diff
@@ -182,14 +182,9 @@ $PREFIX/bin/${CHOST}-gcc -dumpspecs > $specdir/specs
 #   package is installed.
 sed -i -e "/\*link_libgcc:/,+1 s+%.*+& -rpath ${PREFIX}/lib+" $specdir/specs
 
-# Ensure that libgcc_s.so is found in the sysroot. I have done this to mask the fact that
-# strong run_export packages do not get installed into the host prefix (AFAICT) and we
-# should really fix that too. (ping @msarahan)
-cp -f ${PREFIX}/${CHOST}/lib/libgcc_s.so* ${PREFIX}/${CHOST}/sysroot/lib
```

Comment:
Is this conda install issue fixed or did we do some other hack to make this work?

Reply:
Yes, it is fixed. `conda-build` will install strong `run_exports` packages in `host`